src/modules/fancyzones/FancyZonesLib/FancyZones.cpp

```diff
@@ -1049,11 +1067,20 @@ bool FancyZones::ProcessDirectedSnapHotkey(HWND window, DWORD vkCode, bool cycle
     // Check whether Alt is used in the shortcut key combination
     if (GetAsyncKeyState(VK_MENU) & 0x8000)
     {
-        return m_windowMoveHandler.ExtendWindowByDirectionAndPosition(window, vkCode, zoneWindow);
+        bool result = m_windowMoveHandler.ExtendWindowByDirectionAndPosition(window, vkCode, zoneWindow);
+        if (result) {
+            Trace::FancyZones::KeyboardSnapWindowToZone(zoneWindow->ActiveZoneSet());
```

Comment:
Let's keep uniform style :)

```suggestion
        if (result) 
        {
            Trace::FancyZones::KeyboardSnapWindowToZone(zoneWindow->ActiveZoneSet());
```

Reply:
You're right, my bad :) Thank you!